README.md

```diff
@@ -14,3 +15,47 @@ In addition, we are trying to have a well-designed structure ourselves so that v
 
 Through these activities, ONE-vscode can provide a differentiation that cannot be experienced only with the existing CLI by linking the toolchain with the execution environment, ONE Runtime, or a target simulator corresponding to the backend compiler. This will eventually lead to fun and high productivity for developers.
 
+## Getting started
+
+- [How to install?](./docs/HowToInstall.md)
+- [How to use?](./docs/HowToUse.md)
+- [Terms](./docs/Terms.md)
+
+## Extension features
+
+###  One view explorer
+
```

Comment:
How about adding some explanation about 'ONE' icon in left area (activity bar) first? :->

Reply:
I did